### PR TITLE
Spatial Interdictor update: radstorm prot rework, new rods, QoL and efficiency upgrades

### PR DESCRIPTION
**Pull request by Kubius**
[FEAT][REWORK]
## About the PR 

Broad update to interdictors improving their mechanical completeness, quality of options, and consistency. Most important feature is making radiation storm protection reliable, as its mechanism of protection previously meant it would ramp out of practical power consumption ranges in short order.

- [Major Balance/Function] The interdictor's functionality of blocking radiation application from mass radiation now uses a specialized proc to manage the power usage centrally; the power cost per block has dramatically increased, but can only ever be charged once per machine tick, meaning large crowds (human or otherwise) will not cause drastic increases in power draw and a sufficiently supplied interdictor should always block radstorms in their entirety.
- [Balance/Function] Interdictor placement restriction is no longer done based on wires under the unit, and now uses a simpler check; if there are any active interdictors in range that would overlap their fields with the interdictor being activated, activation will fail. This should make placement much more expedient and flexible, while keeping the original intended function of discouraging use of interdictors as barricades.
- [Balance/Function] Interdictor recharge from grid now happens at a variable charge level, instead of constantly requiring the maximum recharge draw; at least ten units of cell power have to be restored in each recharge increment. Small draw functionality, such as from alternate mainboards, will now be much more efficient.
- [Balance/Function] Interdictors now have a basic emag interaction; emagging them strips the access requirement to turn them on and off. Radiation immune emag enjoyers, your moment is now.
- [Balance/Function] Introduced two new specialized interdictor rod variants, Epsilon and Phi. Epsilon provides a massive range (10 radius before any board bonuses), but substantially reduces power efficiency (180% of regular expenditure); Phi provides a minimal range (2 radius before any board bonuses), but improves power efficiency (60% of regular expenditure).
- [Balance] Increased the interdiction distance of the Lambda phase-control rod (cheapest / "default" rod) from 3 to 4 (together with modifiers from board, determines interdiction field radius). Makes "default" interdictors feel a little better rangewise.
- [Balance] Increased Zephyr Field's stamina regen buff from 8 to 12.
- [Balance] Decreased Nimbus recharge efficiency. The rate remains at 75 per cyborg tick, but the power charged to the interdictor is 170% of the amount given to the cyborg. Efficiency rod will return this to nearly 1:1 efficiency.
- [QoL/Balance] Reduced manufacture time of most interdictor rods.
- [QoL/Balance] Interdictor assembly no longer requires a soldering iron; the wire securing step now consists of tightening the wire terminals with screws.
- [Function] Interdictor process no longer unnecessarily uses mult.
- Updated the ingame documentation book to reflect radstorm protection change and current state of assembly process.

## Why's this needed? 

Interdictors' main reason for existing is reliable protection from random events if someone goes to the trouble of building and powering them, and this makes the "reliable" bit actually a thing for radstorms. Other changes improve quality of life in assembling, placing and utilizing interdictors.

Save the chickens!

## Changelog 

```changelog
(u)Kubius
(*)Spatial interdictor mass radiation protection no longer scales with protected individual count, and should reliably weather a whole storm if external power's good.
(*)Spatial interdictors also no longer require a wire beneath them to initialize, but can't initialize in range of another active interdictor.
(+)Spatial interdictors ALSO also can now be emagged (removes access requirement), don't need a soldering iron for assembly anymore (securing wires is now done by screwing down wire terminals), and their electrical draw under low use has been cut down a bit.
```

### CODE REVIEW

File: code/obj/machinery/interdictor.dm [hunk 1]
```diff
@@ -319,15 +390,28 @@
 	throw_range = 5
 	w_class = W_CLASS_NORMAL
 	flags = FPRINT | TABLEPASS | CONDUCT
-	var/interdist = 3
-	//how far the interdictor constructed with this rod will extend its interdiction field
+	var/interdist = 4 //how far the interdictor constructed with this rod will extend its interdiction field
+	var/power_multiplier = null //if present, influences the efficiency of interdictor operation
```
Comment (ZeWaka) [inline code/obj/machinery/interdictor.dm:394]:
```suggestion
	/// how far the interdictor constructed with this rod will extend its interdiction field
	var/interdist = 4
	/// if present, influences the efficiency of interdictor operation
	var/power_multiplier = null 
```

File: code/obj/machinery/interdictor.dm [hunk 2]
```diff
@@ -14,19 +14,26 @@
...
+	//when true, a cost has already been paid for this process, letting radstorm interdict proc know it doesn't need to expend cell charge again
+
 	var/interdict_range = 5 // range of the interdictor's field
 	//for effects that are wide-band interdicted, such as solar flares, this should dictate the response strength
 
 	var/interdict_class = ITDR_STANDARD // type of interdictor
 	//standard interdictors provide only the stellar phenomena protection; alternate variants unlock new functionality
 
+	var/interdict_cost_mult = 1 // interdiction cost multiplier
+	//some part selections can influence this value, raising or lowing the effective energy cost of interdiction activity
```
Comment (ZeWaka) [inline code/obj/machinery/interdictor.dm:35]: doc comments

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/machinery/interdictor.dm (hunk 1 above)
Comment (Kubius) [reply code/obj/machinery/interdictor.dm]: Did differently in [a2c5b2c](https://github.com/goonstation/goonstation/pull/12687/commits/a2c5b2c4369795b21aa94d02747fa63a0b8d19a4).

File: code/obj/machinery/interdictor.dm (hunk 2 above)
Comment (Kubius) [reply code/obj/machinery/interdictor.dm]: Done in [a2c5b2c](https://github.com/goonstation/goonstation/pull/12687/commits/a2c5b2c4369795b21aa94d02747fa63a0b8d19a4).

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**